packages/web/components/MegaMenu/DesktopPlayerStats.tsx

```diff
@@ -41,37 +43,94 @@ export const DesktopPlayerStats: React.FC<PlayerStatsProps> = ({ player }) => {
             _hover={{ transform: 'scale(0.9)' }}
           />
         </MenuButton>
-        <MenuList mt="8px" color="black">
+        <MenuList
+          mt="8px"
+          color="white"
+          boxShadow="2xl"
+          bg="linear-gradient(180deg, rgba(42, 31, 71, 0.9) 6.18%, rgba(17, 3, 32, 0.86) 140%)"
+          borderRadius="md"
+          border={0}
+          px={2}
+          position="relative"
+        >
+          <Icon
+            position="absolute"
+            left="calc(92% - 1.25rem)"
+            top={-3}
+            w={6}
+            h={'auto'}
+            opacity={1}
+            transition="opacity 0.2s"
+            zIndex={2}
+          >
+            <path
+              d="M12 0L24 12C14.6274 12 9.37258 12 0 12L12 0Z"
+              fill="rgba(42, 31, 71, 0.9)"
+            />
+          </Icon>
           <MetaLink
-            color="black"
+            color="white"
             href={getPlayerURL(player) ?? '/'}
             _hover={{ textDecoration: 'none' }}
           >
-            <MenuItem>
+            <MenuItem
+              sx={{
+                '&:hover, &:active, &:focus': {
+                  bg: 'rgba(0,0,0,0.56)',
+                  color: 'white',
+                  borderRadius: 'md',
+                },
+              }}
+            >
```

Comment:
Given this is repeated five times, I'd consider defining a component for it.

Reply:
Agreed. Will make changes after launch if that's cool.